sql/core/pom.xml

```diff
@@ -130,6 +130,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.avro</groupId>
```

Comment:
@srowen .
Maven rejects the newly added test dependency, so I reverted the commit about moving into parent pom. To use different versions, it seems we should keep this here.